tempodb/encoding/vparquet4/block_traceql.go

```diff
@@ -2424,14 +2420,16 @@ func createAttributeIterator(makeIter makeIterFn, conditions []traceql.Condition
 	keyPath, strPath, intPath, floatPath, boolPath string,
 	allConditions bool, selectAll bool,
 ) (parquetquery.Iterator, error) {
+	repLevelOpt := parquetquery.SyncIteratorOptRepetitionLevel(definitionLevel)
```

Comment:
should we name this something more explicit. so we understand we're enabling arrays?

Reply:
Done